src/modules/item/utils.ts

```diff
@@ -328,3 +328,13 @@ export function areSynced(item: Item, entity: DeploymentWithMetadataContentAndPo
   }
   return true
 }
+
+export function buildZipFiles(files: Record<string, Blob | string>, areEqual: boolean) {
```

Comment:
What do you think about renaming this method to something like `buildZipContent`? I was expecting it to work with files, but it's just building the content that it should have.

Reply:
done ✅